utils/xprof_utils.cpp

```diff
@@ -20,6 +20,104 @@ thread_id_t borrow_thread_id(const bt_event *event){
     return bt_field_integer_unsigned_get_value(field);
 }
 
+
+bt_message* create_energy_message(const char* hostname, const process_id_t process_id, const thread_id_t thread_id,
+                                     const uintptr_t hDevice, const uint32_t domain, const uint64_t energy, const uint64_t ts,
+                                     bt_event_class *event_class, bt_self_message_iterator *message_iterator, bt_stream *stream, backend_t backend) {
```

Comment:
```suggestion
bt_message* create_power_message(const char* hostname, const process_id_t process_id, const thread_id_t thread_id,
                                 const uintptr_t hDevice, const uint32_t domain, const uint64_t energy, const uint64_t ts,
                                 bt_event_class *event_class, bt_self_message_iterator *message_iterator, bt_stream *stream, backend_t backend) {
```
Your reading power now